scripts/figma-library-alerts.js

```diff
@@ -36,7 +46,7 @@ module.exports = function (robot) {
     } else {
       // X-Goog- header not present, use default message which will be used in Github Actions
       let data = req.body;
```

Comment:
I would add a check on this as the code will still run even if the body is empty and could lead to some broken looking notifications

Reply:
Good point, changed that 👍